### PR TITLE
Update audit-log-events.md

### PR DESCRIPTION
**Pull request by dingding2**

### CODE REVIEW

File: Teams/audit-log-events.md [hunk 1]
```diff
@@ -181,6 +181,16 @@ If your organization is using the Shifts app in Teams, you can search the audit
 |Added workforce integration|WorkforceIntegrationAdded|The Shifts app is integrated with a third-party system.|
 |Accepted off shift message|OffShiftDialogAccepted|A user acknowledges the off-shift message to access Teams after shift hours.|
 
+## Updates app in Teams activities
+
+If your organization is using the Updates app in Teams, you can search the audit log for activities related to the Updates app. Here's a list of all events that are logged for Updates App activities in Teams in the Microsoft 365 audit log.
```
Comment (JohanFreelancer9) [inline Teams/audit-log-events.md:186]:
```suggestion
If your organization is using the Updates app in Teams, you can search the audit log for activities related to the Updates app. Here's a list of all events that are logged for the Updates app activities in Teams in the Microsoft 365 audit log.
```

File: Teams/audit-log-events.md [hunk 2]
```diff
@@ -181,6 +181,16 @@ If your organization is using the Shifts app in Teams, you can search the audit
...
+|Friendly name|Operation|Description|
+|---|---|---|
+|Create an update request|CreateUpdateRequest|A user successfully creates an update request.|
+|Edite an update request|EditUpdateRequest|A user opens request editing wizard and hits ‘save’ button to confirm and save any changes, or enable/disable the update request directly.|
```
Comment (JohanFreelancer9) [inline Teams/audit-log-events.md:190]:
```suggestion
|Edite an update request|EditUpdateRequest|A user opens the request editing wizard and selects **Save** to confirm and save any changes or enable or disable the update request directly.|
```

File: Teams/audit-log-events.md [hunk 3]
```diff
@@ -181,6 +181,16 @@ If your organization is using the Shifts app in Teams, you can search the audit
...
+|Submit an update|SubmitUpdate|A user successfully submits an update..|
```
Comment (JohanFreelancer9) [inline Teams/audit-log-events.md:191]:
```suggestion
|Submit an update|SubmitUpdate|A user successfully submits an update.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dingding2, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@robmazz 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dingding2** [comment]
Hi @JohanFreelancer9  Thank you very much for helping review this. Do you know when will it take effect in the public doc?

**Comment by JohanFreelancer9** [comment]
> Hi @JohanFreelancer9 Thank you very much for helping review this. Do you know when will it take effect in the public doc?

It is only a pleasure. Your changes will be live once they have been reviewed and merged by the doc author.

**Comment by dingding2** [comment]
Hi @robmazz, could you pls help with a final review?

**Comment by dingding2** [comment]
Gentle ping @robmazz  As we are waiting for this approval before we can release our feature. Thank you very much. CC @scanum

**Comment by scanum** [comment]
Gentle ping @robmazz As we are waiting for this approval before we can release our feature. Thank you very much. CC @scanum

**Comment by robmazz** [comment]
@scanum - couple of things:

- This information is already published (https://learn.microsoft.com/en-us/purview/audit-teams-audit-log-events#updates-app-in-teams-activities)
- This topic is no longer in the Teams repo, it's in the Purview repo with all the other auditing topics

Recommend you close this PR.

**Comment by scanum** [comment]
Sounds good, thanks @robmazz !

**Pull request closed without merging**